doctr/io/pdf.py

```diff
@@ -42,4 +42,4 @@ def read_pdf(
 
     # Rasterise pages to PIL images with pypdfium2 and convert to numpy ndarrays
     with pdfium.PdfDocument(file, password=password) as pdf:
-        return [np.asarray(img) for img in pdf.render_topil(scale=scale, **kwargs)]
+        return [img[0] for img in pdf.render_to(pdfium.BitmapConv.numpy_ndarray, scale=scale, **kwargs)]
```

Comment:
I find the name `img` a bit confusing, because the variable doesn't really store the image, but the result, which is a tuple of the form (image, colour_format). What about renaming it to `result` or doing `img for img, _ in ...`?

Reply:
sec case makes sense :)